src/REPLMode/REPLMode.jl

```diff
@@ -518,6 +518,9 @@ function promptf()
         else
             project_name = projname(project_file)
             if project_name !== nothing
+                if length(project_name) > 20
+                    project_name = project_name[1:20] * "..."
+                end
```

Comment:
Given the ... adds 3 chars, this should take 3 fewer chars than the check length. Also could change to a single elipses char